src/nft-card-back.ts

```diff
@@ -9,7 +9,21 @@ enum TraitType {
   Boost = 'boost',
 }
 
-const TRAIT_HEIGHT = 40
+const TRAIT_HEADER_HEIGHT = 42
+const TRAIT_HEADER_MARGIN_BOTTOM = 8
+
+const RANK_HEIGHT = 40
+const RANK_MARGIN = 8
+
+const traitHeaderStyle = {
+    height: TRAIT_HEADER_HEIGHT + 'px',
```

Comment:
does lit-element not auto-convert numbers to `px` values when you remove 'px' here?

Reply:
No it need the string literal I believe